docs/blog/_posts/2019-12-20-21th-dotty-milestone-release.md

```diff
@@ -0,0 +1,378 @@
+---
+layout: blog-page
+title: Announcing Dotty 0.21.0-RC1
+author: Aggelos Biboudis
+authorImg: /images/aggelos.png
+date: 2019-12-21
+---
+
+Greetings and we wish you Merry Christmas 🎄!
+
+We are excited to announce 0.21.0-RC1 of Scala 3. In this version we add support
```

Comment:
(I would have expected "Dotty" here instead of "Scala 3", until 3.0.0-M1. but, whatevs)